Task2/tests/Task2.2Tests/PolynomialTests.cs

```diff
@@ -0,0 +1,33 @@
+﻿using NUnit.Framework;
+using Task202;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task202.Tests
+{
+	[TestFixture()]
+	public class PolynomialTests
+	{
+		[TestCase(5,4,1,1,5,8,4)]
+		public void GivenOverrideOperatorMultiply_WhenForTwoPozzitivePolynomials_ThenIsPossitive(
+			double firstСoefFirstPolynom, double secondCoefFirstPolynom,
+			double firstСoefSecondPolynom, double secondCoefSecondPolynom,
+			double firstСoefExpectedPolynom, double secondCoefExpectedPolynom, double thirdСoefExpectedPolynom)
+		{
+			//Act
+
+			Polynomial firstPolynom = new Polynomial(new double[2] { firstСoefFirstPolynom, secondCoefFirstPolynom });
+			Polynomial secondPolynom = new Polynomial(new double[2] { firstСoefSecondPolynom, secondCoefSecondPolynom });
+			Polynomial expectedPolynom = new Polynomial(new double[3] { firstСoefExpectedPolynom, secondCoefExpectedPolynom, thirdСoefExpectedPolynom });
```

Comment:
```suggestion
			Polynomial firstPolynom = new Polynomial(new double[] { firstСoefFirstPolynom, secondCoefFirstPolynom });
			Polynomial secondPolynom = new Polynomial(new double[] { firstСoefSecondPolynom, secondCoefSecondPolynom });
			Polynomial expectedPolynom = new Polynomial(new double[] { firstСoefExpectedPolynom, secondCoefExpectedPolynom, thirdСoefExpectedPolynom });
```
размерность тут не нужна